plugins/GSdx/GS.cpp

```diff
@@ -269,7 +269,7 @@ static int _GSopen(void** dsp, const char* title, GSRendererType renderer, int t
 			int w = theApp.GetConfigI("ModeWidth");
 			int h = theApp.GetConfigI("ModeHeight");
 #if defined(__unix__)
-			void *win_handle = (void*)((uptr*)(dsp)+1);
+			void *win_handle = dsp;
```

Comment:
I'm fine with the change, but doesn't that add a warning?

Reply:
The previous code was casting `dsp` in order to perform pointer arithmetic on it (getting the second element of the two-element dynamically typed array pointer that is `pDsp`). Now we want the "first element" or the array pointer itself, so we just store `dsp` and no casting is required.

Reply:
Makes sense, thanks!